packages/@lwc/synthetic-shadow/src/faux-shadow/node.ts

```diff
@@ -428,6 +429,13 @@ defineProperties(Node.prototype, {
         configurable: true,
         writable: true,
     },
+    isConnected: {
+        enumerable: true,
+        configurable: true,
+        get(this: Node) {
+            return isConnected.call(this);
```

Comment:
this `isConnected` is the same you import from `Node.prototype`, I suspect that it must be patched here if it doesn't exist.